execution/engine/eks_engine.go

```diff
@@ -389,12 +388,43 @@ func (ee *EKSExecutionEngine) FetchUpdateStatus(run state.Run) (state.Run, error
 			run.Cpu = &cpu
 			mem := container.Resources.Limits.Memory().ScaledValue(resource.Mega)
 			run.Memory = &mem
+
 		}
 	}
 
 	run, _ = ee.FetchPodMetrics(run)
 	hoursBack := time.Now().Add(-24 * time.Hour)
 
+	events, err := ee.GetEvents(run)
+
+	if err == nil && len(events.PodEvents) > 0 {
+		newEvents := events.PodEvents
+		if run.PodEvents != nil && len(*run.PodEvents) > 0 {
+			priorEvents := *run.PodEvents
+			for _, newEvent := range newEvents {
+				unseen := true
+				for _, priorEvent := range priorEvents {
+					if newEvent.Reason == priorEvent.Reason &&
```

Comment:
perhaps extract this into a `arePodEventsEqual` function for your sanity later on

Reply:
good point, fixed